iron-multi-selectable.html

```diff
@@ -106,6 +116,10 @@
         for (var i = 0; i < selectedItems.length; i++) {
           this._selection.setItemSelected(selectedItems[i], true);
         }
+        // Check for items, since this array is populated only when attached
+        if (this.fallbackSelection && this.items.length && !this._selection.get().length) {
+          this.selectedValues = this._fallbackArray;
```

Comment:
I don't think we can use a fallback array that's generated only when the fallback item changes: if someone causes the fallback to be used and then modifies the array, those modifications become part of `_fallbackArray` until `fallbackSelection` changes.